### ISSUE TITLE
Wallet with subaddresses crashes on sweep_all

### ISSUE THREAD

**Issue by emesik** [monero-project/monero#2781]
```
Monero 'Helium Hydra' (v0.11.0.0-c3c31fa5)
Logging to w2.log
Wallet password: 
Opened wallet: 9wFuzNoQDck1pnS9ZhG47kDdLD1BUszSbWpGfWcSRy9m6Npq9NoHWd141KvGag8hu2gajEwzRXJ4iJwmxruv9ofc2CwnYCE
**********************************************************************
Use "help" command to see the list of available commands.
**********************************************************************
Starting refresh...
Refresh done, blocks received: 1                                
        Account               Balance      Unlocked balance                 Label
       0 9wFuzN       24.923448000000       24.923448000000       Primary account
       1 BZwH58       56.832907280000       33.981591480000      Untitled account
----------------------------------------------------------------------------------
          Total       81.756355280000       58.905039480000
Currently selected account: [0] Primary account
Balance: 24.923448000000, unlocked balance: 24.923448000000
Background refresh thread started
[wallet 9wFuzN]: sweep_all
Wallet password: 
./runwallet.sh: line 6:  8585 Segmentation fault      (core dumped) ~/devel/monero/bin/monero-wallet-cli --testnet --wallet-file $1 --log-level 2 --log-file $1.log

```

The code is from current `master.` Checked on 2 wallets, 100% reproducible. The log file doesn't contain any info on the error.

Any tips on running `gdb` on it or getting details in some other way?

**Comment by stoffu** [comment]
This is due to a bug in the `sweep_all` command which is fixed in #2750.

**Comment by emesik** [comment]
OK, closing then.
